### PR TITLE
User set ignored subs during embedded detection

### PR DESCRIPTION
**Pull request by carrow8993**
Implements a chips dialogue input on the Settings > Subtitles page that allows the user to input their choice of codecs to ignore during embedded sub detection. This is saved to a list as ignored_subs in general settings. Updates indexer logic for both series and movies to check this list during embedded sub check.

### CODE REVIEW

File: bazarr/subtitles/indexer/series.py [hunk 1]
```diff
@@ -41,13 +41,13 @@ def store_subtitles(original_path, reversed_path, use_cache=True):
...
-                                     "ass"):
-                                logging.debug("BAZARR skipping %s sub for language: %s" % (subtitle_codec, alpha2_from_alpha3(subtitle_language)))
-                                continue
+                            skip_flag = False
+                            for sub_codec in settings.general.ignored_subs:
+                                if subtitle_codec.lower() == sub_codec:
+                                    skip_flag = True
+                                    logging.debug("BAZARR skipping %s sub for language: %s" % (subtitle_codec, alpha2_from_alpha3(subtitle_language)))
+                                    break
+                            if skip_flag: continue
```
Comment (anderson-oki) [inline bazarr/subtitles/indexer/series.py:50]: I would say we could use some refactoring here to completely skip this boolean like:

```
if any(subtitle_codec.lower() == sub_codec for sub_codec in settings.general.ignored_subs):
        logging.debug(f"BAZARR skipping {subtitle_codec} sub for language: {alpha2_from_alpha3(subtitle_language)}")
        continue
```

Similar comment also applies to the movies indexer.

**Review by anderson-oki — COMMENTED** [review]
The change is pretty simple, but there are some behavioral changes with Mantine v7 update, can you rebase or merge the `development` branch changes.

File: bazarr/subtitles/indexer/series.py (hunk 1 above)
Comment (carrow8993) [reply bazarr/subtitles/indexer/series.py]: I think I did the rebase/merge/refactor appropriately? I'm sorry, I'm still quite new to this and I managed to break my dev environment so I couldn't do a proper test before heading out of town shortly here.

**Review by anderson-oki — APPROVED** [review]

File: frontend/src/components/forms/SyncSubtitleForm.tsx [hunk 2]
```diff
@@ -126,10 +126,6 @@ const SyncSubtitleForm: FunctionComponent<Props> = ({
 
   const subtitle = selections[0];
 
-  const mediaType = subtitle.type;
-  const mediaId = subtitle.id;
-  const subtitlesPath = subtitle.path;
-
```
Comment (anderson-oki) [inline frontend/src/components/forms/SyncSubtitleForm.tsx:132]: Why this is being removed?

**Review by anderson-oki — CHANGES_REQUESTED** [review]

File: frontend/src/components/forms/SyncSubtitleForm.tsx (hunk 2 above)
Comment (anderson-oki) [reply frontend/src/components/forms/SyncSubtitleForm.tsx:132]: If you are removing this, you gotta fix all its usages in the file.

**Comment by anderson-oki** [comment]
I think there is a small issue with this PR, people have old settings saved on their file, what is going to happen if they don't come to update here?

**Pull request closed without merging**